checker/tests/nullness/flow/PureAndFlow.java

```diff
@@ -50,11 +50,11 @@ void withPure() {
 
     class Cons {
         @org.checkerframework.dataflow.qual.Pure
-        // :: warning: (purity.deterministic.constructor)
+        // :: warning: (purity.deterministic.constructor) :: error: (purity.not.deterministic.not.sideeffectfree.call)
```

Comment:
This is error is happening be the constructor for Object is no longer marked as pure.  Is that intentional?

Reply:
I don't think it makes sense for a constructor to be `@Deterministic`.  In a separate pull request, we should prohibit `@Deterministic` and `@Pure` on constructors.